setup.py

```diff
@@ -8,6 +8,10 @@
 with codecs.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
+# Get requirements from requirements.txt file
+reqs = [line.strip() for line in open('requirements.txt').readlines()]
+requirements = list(filter(None, reqs))
+
 setup(
     # Application name:
     name="nidmviewer",
```

Comment:
We did use version numbers in the requirements for https://github.com/incf-nidash/nidmresults-fsl and so far it works. Is there a way to test if this is fine on NeuroVault?